health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -133,16 +139,33 @@ const ViewHierarchy = () => {
-      };
+                // Assume the first sheet
+                const sheetName = workbook.SheetNames[0];
+                const sheet = workbook.Sheets[sheetName];
+
+                // Convert sheet to JSON and count rows
+                const rows = XLSX.utils.sheet_to_json(sheet);
+
+                // After parsing locally, upload to API
+                const response = await Digit.UploadServices.Filestorage(module, file, tenantId);
+                fileDataTemp.fileStoreId = response?.data?.[0]?.fileStoreId;
+                let fileStoreIdTemp = response?.data?.files?.[0]?.fileStoreId;
```

Biome

[error] This let declares a variable that is only assigned once.

'fileStoreIdTemp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)